include/zmq.h

```diff
@@ -611,6 +611,8 @@ ZMQ_EXPORT void zmq_threadclose (void* thread);
 
 /*  DRAFT Context options                                                     */
 #define ZMQ_MSG_T_SIZE 6
+#define ZMQ_THREAD_AFFINITY 7
```

Comment:
these need to be copied also in src/zmq_draft.h otherwise the no-draft builds will fail